### PR TITLE
Limit guest startup memory size

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1950887

Signed-off-by: Lei Yang <leiayang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Hi @yanan-fu Could you please help me review it? Thanks a lot.

File: qemu/tests/cfg/virtual_nic.cfg [hunk 1]
```diff
@@ -5,6 +5,7 @@
         - private:
             type = virtual_nic_private
             vms = "vm1 vm2 vm3"
+            mem = 4096
```
Comment (yanan-fu) [inline qemu/tests/cfg/virtual_nic.cfg:8]: `mem` in test case will be overwritten by the command line, please use `mem_fixed`, thanks!

Comment (yanglei-rh) [reply qemu/tests/cfg/virtual_nic.cfg]: I have changed it, please help me review it again.Thanks.

**Comment by yanan-fu** [comment]
Could you provide the test result(debug log info), focus on the mem size in the qemu command line, thanks!

**Comment by yanglei-rh** [comment]
Test Pass:
(1/5) repeat1.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (489.44 s)
 (2/5) repeat2.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (490.06 s)
 (3/5) repeat3.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (491.93 s)
 (4/5) repeat4.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (496.80 s)
 (5/5) repeat5.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (492.43 s)
Guest memory:
-m 4096  \

**Comment by yanan-fu** [comment]
Thanks!

**Pull request merged**